core/block/import/objectidgetter.go

```diff
@@ -88,7 +88,11 @@ func (ou *ObjectIDGetter) Get(
 		id := pbtypes.GetString(sn.Snapshot.Data.Details, bundle.RelationKeyId.String())
 		uk, err := domain.UnmarshalUniqueKey(id)
 		if err != nil {
-			return "", treestorage.TreeStorageCreatePayload{}, err
+			id = pbtypes.GetString(sn.Snapshot.Data.Details, bundle.RelationKeyUniqueKey.String())
```

Comment:
Maybe reverse the order of actions? Firstly, check for UniqueKey detail, then try to parse ID. It seems like importing the new version of objects will occur much more often than legacy ones